### PR TITLE
Updated DL links 2.1

### PR DESCRIPTION
**Pull request by mwrightd2**
## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cmastr — APPROVED** [review]

File: pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md [hunk 1]
```diff
@@ -50,16 +50,17 @@ The air-gapped bundle includes the following packages and tooling
 - Docker registry image
 
 1.  Download the air-gapped bundle.
+<p class="message--important"><strong>NOTE: </strong>Enter in the appropriate version in the $VERSION field.</p>
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md:53]: Sorry, my previous suggestion for the VERSION wasn't very clear...  My idea was to add another step to this procedure something like we do here in the docs:   https://docs.d2iq.com/dkp/kommander/2.0/install/networked/#prerequisites, where we tell the user to define a variable we use to eliminate some of repetitive typing later on, and also to make for less things to update when we spin a new version.

So my suggestion would be to do something like this:  (but put the export in the ```sh code block, I couldn't get the suggestion thing to accept nested backtick sections)

```suggestion
Define an environment variable for the Kubernetes version that corresponds with Konvoy release you are installing - you can find the correct Kubernetes version by checking the release notes for the release you are installing.

export VERSION=1.21.6

```

File: pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md [hunk 2]
```diff
@@ -50,16 +50,17 @@ The air-gapped bundle includes the following packages and tooling
...
 
     ```sh
-    curl -O downloads.mesosphere.io/konvoy/airgapped/os-packages_1.21.6_x86_64_rpms.tar.gz
-    curl -O downloads.mesosphere.io/konvoy/airgapped/pip-packages.tar.gz
+    curl -O downloads.d2iq.com/konvoy/airgapped/$VERSION.tar.gz
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md:56]:
```suggestion
    curl -O downloads.d2iq.com/konvoy/airgapped/os-packages_$VERSION_x86_64_rpms.tar.gz
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md [hunk 3]
```diff
@@ -50,16 +50,17 @@ The air-gapped bundle includes the following packages and tooling
...
+    curl -O downloads.d2iq.com/konvoy/airgapped/pip-packages.tar.gz
     ```
 
 1.  Expand the air-gapped bundle artifact.
 
     ```sh
-    tar -xvf os-packages_1.21.6_x86_64_rpms.tar.gz
+    tar -xvf $VERSION.tar.gz
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md:63]:
```suggestion
    tar -xvf os-packages_$VERSION_x86_64_rpms.tar.gz
```

**Review by cneth — CHANGES_REQUESTED** [review]

**Review by Colin1841 — APPROVED** [review]

File: pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md:53]: If we are doing Version for K8s - should we change the env var name to: 

export KUBERNETES_VERSION=1.21.6

And then change that var down below, too?

**Review by cmastr — APPROVED** [review]

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md [hunk 4]
```diff
@@ -50,16 +50,20 @@ The air-gapped bundle includes the following packages and tooling
 - Docker registry image
 
 1.  Download the air-gapped bundle.
+Define an environment variable for the Kubernetes version that corresponds with Konvoy release you are installing - you can find the correct Kubernetes version by checking the release notes for the release you are installing.
+
+export VERSION=1.21.6
```
Comment (dkoshkin) [inline pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/prerequisites/index.md:55]: Don't we want to wrap this in a code snippet "```sh"

**Pull request merged**